parlai/chat_service/services/messenger/agents.py

```diff
@@ -5,33 +5,22 @@
 # LICENSE file in the root directory of this source tree.
 
 import time
-from queue import Queue
 
-from parlai.core.agents import Agent
+from parlai.chat_service.core.agents import ChatServiceAgent
 
 
-class MessengerAgent(Agent):
+class MessengerAgent(ChatServiceAgent):
     """Base class for a person on messenger that can act in a ParlAI world"""
 
     def __init__(self, opt, manager, task_id, messenger_psid, page_id):
```

Comment:
Should this not be `def __init__(self, opt, manager, task_id, receiver_id):`? instead of `messenger_psid`?